cds/modules/migrator/records.py

```diff
@@ -264,6 +264,9 @@ def clean_record(cls, uuid, recid=None):
             record_bucket.bucket.locked = False
             # Make files writable
             for obj in bucket.objects:
+                # skip if file is None (due to a previous soft deletion)
+                if not obj.file:
```

Comment:
Should we be explicit like `obj.file is not None`? Otherwise it looks good to me!

Reply:
what other use case are you thinking of?

Reply:
I don't know for some reason you can set it to empty string by mistake or any other falsy value...but I am totally fine also with that ;)

Reply:
It is a db table relation field, it should not be manipulated manually, but who knows? :)
https://github.com/inveniosoftware/invenio-files-rest/blob/master/invenio_files_rest/models.py#L941

I would leave it like this if you don't have anything against.